deploy/olm-deploy/subscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: namespace-configuration-operator 
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: namespace-configuration-operator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: namespace-configuration-operator.v0.2.0
```

Comment:
please remove this line `startingCSV: namespace-configuration-operator.v0.2.0`

Reply:
@raffaelespazzoli Resolved.